Source/engine/demomode.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file animationinfo.h
+ *
+ * Contains most of the the demomode specific logic
+ */
+#pragma once
+
+#include "miniwin/miniwin.h"
+
+namespace devilution {
+
+class DemoMode {
```

Comment:
Rather than a class with all-static methods, make this a `namespace demo {`.

Reply:
I wasn't sure what is the best way, so I asked @AJenbo but we both were uncertain. So we started with a `class`.
But I'm also happy with a namespace. Fixed. Take a look. 🙂

Reply:
Thanks!